x-pack/plugins/actions/server/builtin_action_types/email.ts

```diff
@@ -61,6 +58,9 @@ const ConfigSchemaProps = {
   secure: schema.nullable(schema.boolean()),
   from: schema.string(),
   hasAuth: schema.boolean({ defaultValue: true }),
+  tenantId: schema.nullable(schema.string()),
+  clientId: schema.nullable(schema.string()),
+  oauthTokenUrl: schema.nullable(schema.string()),
```

Comment:
Is this necessary with the Graph API approach? I didn't specify it and the email worked fine

Reply:
Skipping that validation is a framework level issue (I've opened the [issue](https://github.com/elastic/kibana/issues/112459) to track). Currently we can rely only on the client to validate secrets with config values.